android/festago/app/src/main/java/com/festago/festago/presentation/ui/home/festivallist/FestivalListViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package com.festago.festago.presentation.ui.home.festivallist
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import com.festago.festago.domain.repository.FestivalRepository
+import com.festago.festago.presentation.mapper.toPresentation
+import kotlinx.coroutines.launch
+
+class FestivalListViewModel(
+    private val festivalRepository: FestivalRepository,
+) : ViewModel() {
+    private val _uiState = MutableLiveData<FestivalListUiState>()
+    val uiState: LiveData<FestivalListUiState> = _uiState
+
+    fun loadFestivals() {
+        viewModelScope.launch {
+            _uiState.value = FestivalListUiState.Loading
+            festivalRepository.loadFestivals()
+                .onSuccess {
+                    _uiState.value = FestivalListUiState.Success(
+                        it.map { festival ->
+                            festival.toPresentation()
+                        },
+                    )
+                }.onFailure {
```

Comment:
```suggestion
                }
                .onFailure {
```
작성하신것과 제시드린것중에서 통일성 있게 바꾸기 위해서 저희 팀에는 어떤게 더 어울리는지 확인해보면 좋을거같아요!